modules/codegen/src/smithy4s/codegen/internals/SmithyToIR.scala

```diff
@@ -743,7 +739,18 @@ private[codegen] class SmithyToIR(model: Model, namespace: String) {
     case d: DeprecatedTrait =>
       Hint.Deprecated(d.getMessage.asScala, d.getSince.asScala)
     case doc: DocumentationTrait =>
-      Hint.Documentation(doc.getValue())
+      val memberDocs = shape
+        .members()
+        .asScala
+        .map(_.getTarget())
+        .map(shapeId => (shapeId.name, model.expectShape(shapeId)))
```

Comment:
I think this `shapeId.name` should be the member name instead. Can't see what this outputs as, but I feel like that's going to be a different thing (e.g. in `foo: String` you'll see `String` instead of `foo`).